apps/migrations/0001_initial.py

```diff
@@ -1,8 +1,8 @@
 # Generated by Django 3.2.11 on 2022-09-13 14:02
 
-from django.db import migrations, models
 import django.db.models.deletion
 import tagulous.models.models
+from django.db import migrations, models
```

Comment:
This won't trigger django migrations, right?

Reply:
No. django tracks already applied migrations using the django_migrations table.